components/dashboard/src/workspaces/Workspaces.tsx

```diff
@@ -183,10 +114,92 @@ export default class Workspaces extends React.Component<WorkspacesProps, Workspa
+                <>
+                    <div className="lg:px-28 px-10 py-2 flex">
+                        <div className="flex">
+                            <div className="py-4">
+                                <svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 16 16" width="16" height="16"><path fill="#A8A29E" d="M6 2a4 4 0 100 8 4 4 0 000-8zM0 6a6 6 0 1110.89 3.477l4.817 4.816a1 1 0 01-1.414 1.414l-4.816-4.816A6 6 0 010 6z" /></svg>
+                            </div>
+                            <input type="search" className="text-sm" placeholder="Search Workspaces" onChange={(v) => { if (workspaceModel) workspaceModel.setSearch(v.target.value) }} />
+                        </div>
+                        <div className="flex-1" />
+                        <div className="py-3">
+                        </div>
+                        <div className="py-3 pl-3">
+                            <DropDown prefix="Limit: " contextMenuWidth="w-32" activeEntry={workspaceModel ? workspaceModel?.limit + '' : undefined} entries={[{
+                                title: '50',
+                                onClick: () => { if (workspaceModel) workspaceModel.limit = 50; }
+                            }, {
+                                title: '100',
+                                onClick: () => { if (workspaceModel) workspaceModel.limit = 100; }
+                            }, {
+                                title: '200',
+                                onClick: () => { if (workspaceModel) workspaceModel.limit = 200; }
+                            }]} />
+                        </div>
+                        <button onClick={showStartWSModal} className="ml-2">New Workspace</button>
+                    </div>
+                    <ItemsList className="lg:px-28 px-10">
+                        <div className="border-t border-gray-200 dark:border-gray-800"></div>
+                        {
+                            activeWorkspaces.map(e => {
+                                return <WorkspaceEntry key={e.workspace.id} desc={e} model={workspaceModel} stopWorkspace={wsId => getGitpodService().server.stopWorkspace(wsId)} />
+                            })
+                        }
+                        {
+                            activeWorkspaces.length > 0 && <div className="py-6"></div>
+                        }
+                        {
+                            inactiveWorkspaces.length === 0 ? null :
+                                <Item className="w-full bg-gray-50 py-3 px-3">
+                                    <ItemField className=" flex flex-col">
+                                        <div className="text-gray-400 text-sm text-center">Unpinned workspaces that have been inactive for more than 14 days will be automatically deleted. <a className="gp-link" href="https://www.gitpod.io/docs/life-of-workspace/#garbage-collection">Learn more</a></div>
```

Comment:
**issue**: Center-aligned text here looks good! However, this certainly does not need a hover state as it makes users feel this is an actionable or clickable element. But this is out of the scope of the changes of this PR as it affects the `ItemField` component.